ImputationPipeline/notebooks/CreateManuallyQCdPRSResultsFile.ipynb

```diff
@@ -27,29 +27,67 @@
+    "                                                        style = {\"description_width\":'initial'},\n",
+    "                                                        layout = widgets.Layout(width='auto')\n",
+    "                                                       )\n",
+    "        self.table_selection_dropdown.observe(self.select_table, names = \"value\")\n",
+    "        display(self.table_selection_dropdown)\n",
+    "        \n",
+    "    def select_table(self, change):\n",
+    "        if change['old'] != None:\n",
+    "            self.app.close_widgets()\n",
+    "        if change['new'] != None:\n",
+    "            self.app = ResultsModificationGUI(self.workspace_namespace, self.workspace_name, self.workspace_bucket_name, change['new'])\n",
+    "            self.app.run()\n",
     "\n",
     "class ResultsModificationGUI:\n",
     "    eastern_tz = pytz.timezone('US/Eastern')\n",
     "    \n",
-    "    def __init__(self, workspace_namespace, workspace_name, workspace_bucket_name):\n",
+    "    def __init__(self, workspace_namespace, workspace_name, workspace_bucket_name, table_name):\n",
     "        self.workspace_namespace = workspace_namespace\n",
     "        self.workspace_name = workspace_name\n",
     "        self.workspace_bucket_name = workspace_bucket_name\n",
+    "        self.table_name = table_name\n",
     "        self.lab_batch_output_box = widgets.Output(layout={'border': '1px solid black'})\n",
     "        self.status_output_box = widgets.Output(layout={'border': '1px solid black'})\n",
-    "        display(self.status_output_box)\n",
-    "        display(self.lab_batch_output_box)\n",
+    "        self.selected_batch_gui = None\n",
+    "        self.registered_widgets = []\n",
+    "        self.register_widget(self.status_output_box)\n",
+    "        self.register_widget(self.lab_batch_output_box)\n",
     "        self.initialize_workspace_info_and_gcloud()\n",
     "\n",
     "        \n",
     "    def run(self):\n",
     "        self.build_lab_batch_selection_section()\n",
     "        \n",
+    "    def close_widgets(self):\n",
+    "        for widget in self.registered_widgets:\n",
+    "            widget.close()\n",
+    "        if self.selected_batch_gui != None:\n",
+    "            self.selected_batch_gui.close_widgets()\n",
+    "        \n",
+    "    def register_widget(self, widget):\n",
+    "        self.registered_widgets.append(widget)\n",
+    "        display(widget)\n",
+    "        \n",
     "    def initialize_workspace_info_and_gcloud(self):\n",
     "        with self.status_output_box:\n",
-    "            print(\"Getting sample_set information for workspace \" + self.workspace_namespace + \"/\" + self.workspace_name + \"...\")\n",
+    "            print(f\"Getting {self.table_name} information for workspace \" + self.workspace_namespace + \"/\" + self.workspace_name + \"...\")\n",
```

Comment:
This is literally the only thing that I could find, which would be using the `{}` brackets instead of the `+` for this string